Scripts/restoreazuredb.ps1

```diff
@@ -1,49 +1,93 @@
 param (
-    [parameter(Mandatory=$true)]
-    [string]$dbname,
-
-    [parameter(Mandatory=$true)]
-    [string]$Rgroup,
+    # [parameter(Mandatory=$true)]
+    # [string]$dbname,
 
     [parameter(Mandatory=$true)]
     [string]$servername,
```

Comment:
Server Name is derived. See Mohit code and Wiki.

Further than that. Look at Mohit's code in aws_stack_provision.ps1 and model your code around that, and keeping my comments in mind - his is not perfect yet. The model you have here is not correct. 
See PR https://github.com/lansa/db-regression/pull/13

Reply:
Will change that to derived for azure sql server